test/test_dynamic_shapes.py

```diff
@@ -178,6 +178,37 @@ def test_expand_symint_correctness(self):
     self.assertEqual(t3.shape[0], 2)
     self.assertEqual(expand_out_aten.cpu(), expand_out_xla.cpu())
 
+  def test_topk_symint(self):
+    dev = xm.xla_device()
+    t1 = torch.zeros([5, 2], device=dev)
+    t1[3][0] = 1
+    t1[3][1] = 1
+    t2 = torch.nonzero(t1)
+    t3 = torch.zeros([10, 2], device=dev)
+    values, indices = torch.topk(t3, t2.shape[0], dim=0)
```

Comment:
I wonder if your change can handle the case where t3 is dynamic, eg t3=torch.nonzero(t1).

Reply:
I think this feature is already supported by xla through dynamic padder (static topk also works with dynamic t3). I'll add a test case for it later.